lib/importmap/npm.rb

```diff
@@ -86,7 +86,7 @@ def get_json(uri)
     end
 
     def find_latest_version(response)
-      latest_version = response.dig('dist-tags', 'latest')
+      latest_version = response.is_a?(String) ? response : response.dig('dist-tags', 'latest')
```

Comment:
The response may be blank. One reported issue (https://github.com/rails/importmap-rails/issues/245#issue-2127272496) made me question this possibility.

If this is the case then a fallback may be worthwhile. I am writing it with available Rails methods, but can change it to use Ruby-only accessible ones.

```suggestion
      latest_version = response.is_a?(String) ? (response.presence || 'unknown') : response.dig('dist-tags', 'latest')
```